scripts/node/icons/icons.config.js

```diff
@@ -27,9 +27,9 @@ module.exports = {
       lib: `${PATH_ICONS_LIB}/png`,
     },
     react: {
-      src: `${ICONS_ROOT}/svg`,
+      src: `${PATH_ICONS_LIB}/svg`,
```

Comment:
reading from `lib/svg` instead of from `src/icons/` allows us to skip the extra optimization step for building components